management/server/http/api/openapi.yml

```diff
@@ -375,7 +375,7 @@ components:
           type: boolean
           example: false
         auto_groups:
-          description: Setup key groups to auto-assign to peers registered with this key
+          description: List of group IPs to auto-assign to peers registered with this key
```

Comment:
Considering clarity, would it be better to rename the` group IPs` to `group names`?